Tasks/e0_breadth_first_search.py

```diff
@@ -10,5 +10,13 @@ def bfs(g: nx.Graph, start_node: Hashable) -> List[Hashable]:
 	:param start_node: starting node for search
 	:return: list of nodes in the visited order
 	"""
-	print(g, start_node)
-	return list(g.nodes)
+	visited = [start_node]
+	q.enqueue(start_node)
+	while q.leng() > 0:
+		start_node = q.dequeue()
+		neighbors = list(g.neighbors(start_node))
```

Comment:
```suggestion
		neighbors = g.neighbors(start_node)
```
Do not need to cast to list if iterate it

Reply:
non blocking